packages/inventory/src/utils/query.ts

```diff
@@ -12,14 +14,19 @@ export function getListQuery(
   config: FindConfig<InventoryItem> = { relations: [], skip: 0, take: 10 }
 ) {
   const inventoryItemRepository = manager.getRepository(InventoryItem)
-  const query = buildQuery(selector, config)
 
-  const queryBuilder = inventoryItemRepository.createQueryBuilder("inv_item")
+  const { q, ...selectorRest } = selector
+  const query = buildQuery(selectorRest, config) as ExtendedFindConfig<InventoryItem> & {
```

Comment:
we need to rethink the casting, if it can be avoided. Or atleast, create a new type that these queries conform to.